modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -280,6 +281,16 @@ exports.send = function(req, res) {
       });
     },
 
+    // Here we collect some message data into the influxdb
+    function (message, done) {
+      // the module returns a Promise
+      // it is async, but we'll let it run wild (not waiting for the promise to
+      // be fulfilled or rejected)
+      messageToInfluxService(message);
```

Comment:
This is as far as I've made it in my review today. Friend just arrived, will try to look through the rest later.

@mrkvon Thanks for all the effort you've put into this.